tests/PHPUnit/Domain/Service/Precondition/ComposerIsAvailableUnitTest.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+namespace PhpTuf\ComposerStager\Tests\PHPUnit\Domain\Service\Precondition;
+
+use PhpTuf\ComposerStager\Domain\Exception\IOException;
+use PhpTuf\ComposerStager\Domain\Service\Precondition\ComposerIsAvailable;
+use PhpTuf\ComposerStager\Domain\Value\Path\PathInterface;
+use PhpTuf\ComposerStager\Infrastructure\Service\Finder\ExecutableFinderInterface;
+use PhpTuf\ComposerStager\Tests\PHPUnit\TestCase;
+
+/**
+ * @coversDefaultClass \PhpTuf\ComposerStager\Domain\Service\Precondition\ComposerIsAvailable
+ *
+ * @covers \PhpTuf\ComposerStager\Domain\Service\Precondition\ComposerIsAvailable::__construct
+ *
+ * @uses \PhpTuf\ComposerStager\Domain\Service\Precondition\AbstractPrecondition
+ *
+ * @property \PhpTuf\ComposerStager\Domain\Value\Path\PathInterface|\Prophecy\Prophecy\ObjectProphecy $activeDir
+ * @property \PhpTuf\ComposerStager\Domain\Value\Path\PathInterface|\Prophecy\Prophecy\ObjectProphecy $stagingDir
+ * @property \PhpTuf\ComposerStager\Infrastructure\Service\Finder\ExecutableFinderInterface|\Prophecy\Prophecy\ObjectProphecy $executableFinder
+ */
+class ComposerIsAvailableUnitTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        $this->activeDir = $this->prophesize(PathInterface::class);
+        $this->stagingDir = $this->prophesize(PathInterface::class);
+        $this->executableFinder = $this->prophesize(ExecutableFinderInterface::class);
+        $this->executableFinder
```

Comment:
This method prophecy seems incomplete. Should we have some sort of `shouldBeCalledOnce()` or something? Or is it just detritus that should be removed?

Reply:
Removed.